### PR TITLE
Show disease name in IR tables with count

### PR DESCRIPTION
**Pull request by jgoley**
Shows counts and disease like so:
screen shot 2017-03-30 at 5 55 01 pm

[PT story](https://www.pivotaltracker.com/story/show/142404863)

### CODE REVIEW

File: imports/ui/helpers.coffee [hunk 1]
```diff
@@ -11,8 +11,8 @@ UI.registerHelper 'formatDateRange', (dateRange)->
   return formatDateRange(dateRange)
 
 UI.registerHelper 'incidentToText', (incident) ->
-  if @cases
-    incidentDescription = pluralize("case", @cases)
+  if incident.cases
```
Comment (nathanathan) [inline imports/ui/helpers.coffee]: Why does the variable for the cases property need to change, but not the variable for the deaths and specify properties?

File: server/methods.coffee [hunk 2]
```diff
@@ -177,6 +177,7 @@ Meteor.methods
     options.publishDate = source.publishDate
     incidents = createIncidentReportsFromEnhancements(enhancements, options)
     incidents = incidents.map (incident) ->
+      console.log incident
```
Comment (nathanathan) [inline server/methods.coffee]: Could you remove this?

File: imports/ui/helpers.coffee (hunk 1 above)
Comment (jgoley) [inline imports/ui/helpers.coffee]: Oh that must have been incidental.
I also didn't realize we already had a helper registered. I'll combine `incidentTypeWithCountAndDisease` with this.

**Comment by nathanathan** [comment]
👍

**Pull request merged**